### DISCUSSION TITLE
Can nicegui adjust its default apperance?

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1210] [Ideas / Feature Requests]
Could all the widget align to the center or use the wide mode like streamlit?
With streamlit the defaut apperance is very good,we almost don't need to adjust it ,but if we want to adjust something ,for example ,the ability of row() in the nicegui, It's a little hard to implement in streamlit.  nicegui is  more powerfull to adjust the attribute of  widgets,but we need much time to adjust the display effects.... 

here is two picture 
![微信截图_20230721045512](https://github.com/zauberzeug/nicegui/assets/25096045/512cddae-c5c2-4bf9-a73b-f13fac4f1d83)
![微信截图_20230721045530](https://github.com/zauberzeug/nicegui/assets/25096045/e1388fa1-8d00-4000-8fd2-7f5f3c7cfcc4)

**Comment by falkoschindler** [comment]
It's not clear to me what _exactly_ you're looking for. But here are my thoughts on NiceGUI's default layout and style:
Since NiceGUI is based on Quasar, the look-and-feel is very similar. And because NiceGUI is more generic than Streamlit, it's harder to automatically create a better layout, since it's not clear what the user wants. While Streamlit only has a sidebar and only wide or narrow mode, NiceGUI has tons of options. So even changing the default padding might look weird in certain situations.

**Comment by rodja** [comment]
Also, if we would apply certain attributes as standard, these would need to be changed/updated/removed for customization (and hence users need to be aware of them). We have therefore tried to apply only a minimum of default styling options.

**Reply by v479038280** [reply]
It's better to be able to keep its customization ,althouth it take some time to adjust it to look better(which almost dont need to adjust in streamlit ),with the use of props and classes nicegui is powerfull.
